src/darwin/Framework/CHIP/MTRDeviceControllerDelegate.h

```diff
@@ -19,25 +19,26 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-typedef NS_ENUM(NSUInteger, MTRPairingStatus) {
-    MTRPairingStatusUnknown = 0,
-    MTRPairingStatusSuccess = 1,
-    MTRPairingStatusFailed = 2,
-    MTRPairingStatusDiscoveringMoreDevices = 3
-};
+typedef NS_ENUM(NSUInteger, MTRCommissioningStatus) {
```

Comment:
Does this need to be unsigned? I think the "default" backing type for `NS_ENUM` is usually `NSInteger`

Reply:
The underlying SDK type is uint8_t.  It probably does not matter too much whether the type here is unsigned or signed.... That said, we never plan to have negative values here, so unsigned is conceptually clearer....

Reply:
Changed to NSInteger.